src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are you need them?

Reply:
I tested my code with main class and I needed them 
I didn't write .gitignor correctly and everything was pushed